jreleaser.yml

```diff
@@ -0,0 +1,46 @@
+project:
+  name: spoon-core
+  description: A plugin to lock and validate the integrity of the dependencies of a maven based project.
+  longDescription: Spoon is an open-source library to analyze, rewrite, transform, transpile Java source code. 
+                   It parses source files to build a well-designed AST with powerful analysis and transformation API. 
+                   It supports modern Java versions up to Java 20.
+  authors:
+    - slarse
+    - monperrus
+    - nharrand
+    - martinwitt
+    - sirywell
+    - I-Al-Istannen
+  license: (MIT OR CECILL-C)
+  links:
+    homepage: https://github.com/chains-project/maven-lockfile
```

Comment:
Not quite

Reply:
Do we want the github page or [the spoon page](https://spoon.gforge.inria.fr/) here?

Reply:
I would say spoon page